src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
```

Comment:
Have you got `IsoscelesTrapezoid` figure at least one time? 
I think no (you can read documentation of `nextInt` method and that `nextInt(2)` will never return 2)